### PR TITLE
ADD: Kill Batch

### PR DESCRIPTION
**Pull request by sihrc**

### CODE REVIEW

File: R/api_handler.R [hunk 1]
```diff
@@ -19,6 +19,8 @@ make_request <- function(data, api, api_key = FALSE, cloud = FALSE, batch = FALS
     api_key <- .indicoio$api_key
   }
 
+  batch <- typeof(data) == "list" || length(data) > 1
```
Comment (madisonmay) [inline R/api_handler.R]: I don't think this is a valid check.  Wouldn't length(string) evaluate to True here?

Comment (sihrc) [inline R/api_handler.R]: oh does it not do (typeof(data) == "list") || (length(data) > 1)

Comment (madisonmay) [inline R/api_handler.R]: If the first section evaluates to false, the second bit may still trigger batch to be set to true.

Comment (sihrc) [inline R/api_handler.R]: Which is what we want. Because if it's not a list then it could be a vector that could have length > 1

Comment (sihrc) [inline R/api_handler.R]: Type checking for vector is awful in that even Strings pass as vectors. Length of Strings when called like that is 1

Comment (madisonmay) [inline R/api_handler.R]: Wut.  Ok, I'll trust you, but that is incredibly weird behavior as far as I'm concerned.

File: R/api_handler.R [hunk 2]
```diff
@@ -90,8 +92,10 @@ format_image <- function(img, size) {
       img <- base64decode(img)
       img <- readPNG(img)
     }
-  } else if (is.matrix(img) || is.data.frame(img)) {
-    warning("Image input as matrices and dataframes will be deprecated in the next major release");
+  } else if (is.vector(img)) {
```
Comment (madisonmay) [inline R/api_handler.R]: We use is.vector to check for batch inputs here, but `typeof(data) == "list" || length(data) > 1` in `make_request`?  What's the rationale here @sihrc?

Comment (sihrc) [inline R/api_handler.R]: oh, need to change this one.

File: R/api_handler.R (hunk 1 above)
Comment (sihrc) [inline R/api_handler.R]: screen shot 2015-07-30 at 9 58 04 pm

**Comment by madisonmay** [comment]
Looks great to me.  Merging!

**Comment by madisonmay** [comment]
Wait, not so fast!  One edit to make -- we still need to export the batch functions.

File: NAMESPACE [hunk 3]
```diff
@@ -1,19 +1,5 @@
 # Generated by roxygen2 (4.1.1): do not edit by hand
 
-export(batch_content_filtering)
```
Comment (madisonmay) [inline NAMESPACE]: We need these lines -- can we regenerate this file?

**Pull request merged**